camel-example-spring-boot-apm-opentracing/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.camel.springboot.example</groupId>
+        <artifactId>examples</artifactId>
+        <version>3.5.0</version>
+    </parent>
+
+    <artifactId>camel-example-spring-boot-apm-opentracing</artifactId>
+    <packaging>pom</packaging>
+    <name>Camel SB Examples :: OpenTracing APM</name>
+    <description>An example showing how to trace incoming and outgoing messages from Camel with OpenTracing with ElastiCo APM
+    </description>
+
+    <properties>
+        <category>Management and Monitoring</category>
+        <title>OpenTracing APM</title>
+        <spring.boot-version>${spring-boot-version}</spring.boot-version>
+    </properties>
+
+
+    <!-- import Spring-Boot and Camel BOM -->
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${spring.boot-version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.camel.springboot</groupId>
+                <artifactId>camel-spring-boot-dependencies</artifactId>
```

Comment:
there is a new camel-spring-boot-bom that is maybe better to use

Reply:
Can you please clarify

Reply:
https://camel.apache.org/camel-spring-boot/latest/#_camel_spring_boot_bom_vs_camel_spring_boot_dependencies_bom

Reply:
https://repo.maven.apache.org/maven2/org/apache/camel/springboot/camel-spring-boot-dependencies/ 
3.5.0 is latest in maven

Reply:
camel-spring-boot-bom